src/Header/Nav/__snapshots__/Nav.test.jsx.snap

```diff
@@ -10,77 +10,90 @@ exports[`Nav Matches snapshot with multiple accounts links 1`] = `
   >
     My account
   </h2>
-  <ul
-    className="menu"
-  >
-    <li
-      key="https://accounts.nice.org.uk/profile"
-    >
-      <a
-        href="https://accounts.nice.org.uk/profile"
-      >
-        Joe Bloggs
-      </a>
-    </li>
-    <li
-      key="https://accounts.nice.org.uk/signout"
-    >
-      <a
-        href="https://accounts.nice.org.uk/signout"
-      >
-        Sign out
-      </a>
-    </li>
-  </ul>
-</nav>
-`;
-
-exports[`Nav Matches snapshot with no accounts links 1`] = `
-<div
-  className="wrapper"
-  id="header-menu"
->
-  <nav
-    className="nav"
+  <div
+    className="menuWrapper"
   >
     <ul
-      aria-labelledby="header-menu-button"
-      className="menu"
+      className="menuList"
       role="menu"
     >
       <li
-        key="link1"
+        key="https://accounts.nice.org.uk/profile"
         role="presentation"
       >
         <a
-          aria-current={null}
-          href="https://url1/"
+          href="https://accounts.nice.org.uk/profile"
           role="menuitem"
         >
-          <span>
-            First link
-          </span>
+          Joe Bloggs
         </a>
       </li>
       <li
-        key="link2"
+        key="https://accounts.nice.org.uk/signout"
         role="presentation"
       >
         <a
-          aria-current={null}
-          href="https://url2/"
+          href="https://accounts.nice.org.uk/signout"
           role="menuitem"
         >
-          <span>
-            <abbr
-              title="Abbreviation title"
-            >
-              Second link
-            </abbr>
-          </span>
+          Sign out
         </a>
       </li>
     </ul>
+  </div>
+</nav>
+`;
+
+exports[`Nav Matches snapshot with no accounts links 1`] = `
+<div
+  className="wrapper"
+  id="header-menu"
+>
+  <nav
+    className="nav"
+  >
+    <div
+      className="menuWrapper"
+    >
+      <ul
+        aria-labelledby="header-menu-button"
+        className="menuList"
+        role="menu"
+      >
+        <li
+          key="link1"
+          role="presentation"
+        >
+          <a
+            aria-current={null}
+            href="https://url1/"
```

Comment:
Any particular reason why there's a trailing slash?

Reply:
Just because URLs copied from the browser always have a trailing slash on the end. It's set in ___mocks__/links.json so could be easily changed